test/tests/functional/pbs_equiv_classes.py

```diff
@@ -1667,10 +1672,35 @@ def test_multiple_job_preemption_order(self):
 
         # Only One equivalence class for jid2 and jid3 is present since both
         # suspended jobs are of same type and running on same vnode
-        t = int(time.time())
+
         self.scheduler.log_match("Number of job equivalence classes: 2",
                                  max_attempts=10, starttime=self.t)
 
+        # Add a job to Queue state
+        a = {'Resource_List.ncpus': 1}
+        J = Job(TEST_USER, attrs=a)
+        jid6 = self.server.submit(J)
+        self.server.expect(JOB, {'job_state': 'Q'}, id=jid6)
+
+        # Set scheduling = false
```

Comment:
Remove this comment.  It doesn't say anything that the manager() call doesn't already.